client-app/shared/catalog/components/category/filters-popup-sidebar.vue

```diff
@@ -13,26 +13,11 @@
     >
       <template #prepend="{ loading: _loading }">
         <div class="filters-popup-sidebar__container">
-          <div v-if="!hideSorting" class="filters-popup-sidebar__sorting">
-            <span class="filters-popup-sidebar__sorting-label">
-              {{ $t("pages.catalog.sort_by_label") }}
-            </span>
-
-            <VcSelect
-              v-model="sortQueryParam"
-              size="sm"
-              text-field="name"
-              value-field="id"
-              :disabled="_loading"
-              :items="PRODUCT_SORTING_LIST"
-            />
-          </div>
-
           <VcCheckbox
             v-if="!hideControls"
             :model-value="popupSidebarFilters.inStock"
             class="filters-popup-sidebar__control"
-            :disabled="loading"
+            :disabled="_loading"
```

Comment:
why _loading
rename to somethingloading

Reply:
It's just for small template context, not for all of component